code/_onclick/hud/screen_objects.dm

```diff
@@ -23,6 +23,8 @@
 	master = null
 	return ..()
 
+/obj/screen/orbit(atom/A, radius = 10, clockwise = FALSE, rotation_speed = 20, rotation_segments = 36, pre_rotation = TRUE, lockinorbit = FALSE)

```

Comment:
I think you don't need to specify the arguments if you're just going to ignore them.